slasher/flags/flags.go

```diff
@@ -21,4 +21,21 @@ var (
 		Name:  "tls-key",
 		Usage: "Key for secure gRPC. Pass this and the tls-cert flag in order to use gRPC securely.",
 	}
+	// BeaconCert defines a flag for the beacon api certificate.
+	BeaconCert = cli.StringFlag{
+		Name:  "beacon-tls-cert",
+		Usage: "Certificate for secure beacon gRPC connection. Pass this in order to use becon gRPC securely.",
+	}
+	// BeaconProvider defines a flag for the beacon host ip or address.
+	BeaconProvider = cli.StringFlag{
+		Name:  "beacon-provider",
```

Comment:
Can you use the same flag names as the validator client?

Reply:
changed, thanks